website/react-magma-docs/src/pages/api/tag.mdx

```diff
@@ -0,0 +1,307 @@
+
+```tsx
+import React from 'react';
+import { Tag } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <>
+      <Tag>Size Default</Tag>
+      <Spacer axis={SpacerAxis.horizontal} size={magma.spaceScale.spacing03} />
+      <Tag size="small">Size Small</Tag>
+    </>
+  );
+}
+```
+
+## Disabled
+
+```tsx
+import React from 'react';
+import { Spacer, Tag } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <>
+      <Tag disabled>Disabled</Tag>
+      <Spacer axis={SpacerAxis.horizontal} size={magma.spaceScale.spacing03} />
+      <Tag disabled color="lowContrast">
+        Disabled Low Contrast
+      </Tag>
+    </>
+  );
+}
+```
+
+## Inverse
+
+`isInverse` is an optional boolean prop, that may be passed in when the component is to be displayed on a dark background.
+
+```tsx
+import React from 'react';
+import { Card, CardBody, Tag } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse>
+      <CardBody>
+        <Tag isInverse>Inverse Default</Tag>
+        <Spacer
+          axis={SpacerAxis.horizontal}
+          size={magma.spaceScale.spacing03}
+        />
+        <Tag isInverse color="primary">
+          Inverse Primary
+        </Tag>
+        <Spacer
+          axis={SpacerAxis.horizontal}
+          size={magma.spaceScale.spacing03}
+        />
+        <Tag isInverse color="lowContrast">
+          Inverse Low Contrast
+        </Tag>
+        <Spacer
+          axis={SpacerAxis.horizontal}
+          size={magma.spaceScale.spacing03}
+        />
+        <Tag isInverse color="highContrast">
+          Inverse High Contrast
+        </Tag>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Inverse Disabled
+
+```tsx
+import React from 'react';
+import { Card, CardBody, Spacer, Tag } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <Card isInverse>
+      <CardBody>
+        <Tag isInverse disabled>
+          Inverse Disabled
+        </Tag>
+        <Spacer
+          axis={SpacerAxis.horizontal}
+          size={magma.spaceScale.spacing03}
+        />
+        <Tag isInverse disabled color="lowContrast">
+          Inverse Disabled Low Contrast
+        </Tag>
+      </CardBody>
+    </Card>
+  );
+}
+```
+
+## Internationalization
+
+If you're using strings in your component, they must be internationalized.
+
+Full example of <Link to="/api/internationalization">internationalization override options</Link>
+
+```tsx
+import React from 'react';
+import { es } from 'date-fns/locale';
+import { DatePicker, I18nContext, defaultI18n } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <I18nContext.Provider
+      value={{
+        ...defaultI18n,
+        example: 'example i18n',
+      }}
+    >
+      <tag>override default i18n value: </tag>
+    </I18nContext.Provider>
+  );
+}
+```
+
+## Tag Props
+
+**Any other props supplied will be provided to the wrapping `div` element.**
+
+<DeleteableTagProps />
```

Comment:
Above each of the `props` tables put `### Deletable Tag Props` or `### Clickable Tag Props` so they know what they are looking at.